.travis.yml

```
@@ -16,18 +16,14 @@ env:
 
 matrix:
   include:
-    - php: 5.6
-      env: DB=MYSQL PHPCS_TEST=1 PHPUNIT_TEST=1
-    - php: 7.0
-      env: DB=PGSQL PHPUNIT_TEST=1
     - php: 7.1
-      env: DB=MYSQL PHPUNIT_COVERAGE_TEST=1
+      env: DB=PGSQL INSTALLER_VERSION=4.5.x-dev PHPCS_TEST=1 PHPUNIT_TEST=1
     - php: 7.2
-      env: DB=MYSQL PHPUNIT_TEST=1
-    - php: 7.2
-      env: DB=MYSQL BEHAT_TEST=1
+      env: DB=MYSQL INSTALLER_VERSION=4.6.x-dev BEHAT_TEST=1
     - php: 7.3
-      env: DB=MYSQL PHPUNIT_TEST=1 NPM_TEST=1
+      env: DB=MYSQL INSTALLER_VERSION=4.6.x-dev PHPUNIT_COVERAGE_TEST=1 NPM_TEST=1
+    - php: 7.4
+      env: DB=MYSQL INSTALLER_VERSION=4.x-dev PHPUNIT_TEST=1 NPM_TEST=1
```

We should only need to run NPM tests once.

```suggestion
      env: DB=MYSQL INSTALLER_VERSION=4.x-dev PHPUNIT_TEST=1
```

---

Updated